src/material-experimental/mdc-progress-spinner/progress-spinner.scss

```diff
@@ -0,0 +1,18 @@
+@import '@material/circular-progress/mixins.import';
+@import '../mdc-helpers/mdc-helpers';
+
+
+@include mdc-circular-progress-core-styles($query: $mat-base-styles-without-animation-query);
+
+.mat-mdc-progress-spinner {
```

Comment:
Can you add comments for each style that explains why it's necessary on top of MDC's styles?

Reply:
For some reason the spinners are getting a style that sets the height and width to 48px by default and it seems like its coming from the MDC side.

Reply:
👍 we just want to include comments in the code any time we add CSS on top of MDC so that we remember _why_ those styles are necessary. This will help us justify to the material team why they exist and help us figure out how to get rid of them over time